File: problem056/solution.py
from src import helpers

max = 0
for a in range(1, 100 + 1):
    for b in range(1, 100 + 1):
        c = sum(int(x) for x in str(a ** b))
        if c > max:
            max = c
helpers.solution(max)

File: problem056/solution.php
<?php
$start = microtime(true);
require __dir__ . "/../src/helpers.php";

$max = 0;
for ($a = 1; $a <= 100; $a++) {
    for ($b = 1; $b <= 100; $b++) {
        $sum = array_sum(str_split(bcpow($a, $b)));
        if ($sum > $max) {
            $max = $sum;
        }
    }
}
solution($max);
duration($start);
